roles/tunnel-server/templates/site-https.conf.j2

```diff
@@ -63,23 +63,26 @@ server {
     proxy_set_header  X-Real-IP  $remote_addr;
     proxy_set_header  X-Forwarded-For $proxy_add_x_forwarded_for;
     proxy_set_header  X-Original-Host $http_host;
-    proxy_set_header  Host {{ localhost_target_default }};
-    proxy_redirect    http://{{ localhost_target_default }}/ http://$http_host/;
+    proxy_set_header  Host {{ services[0].name }}.test;
+    proxy_redirect    http://{{ services[0].name }}.test/ http://$http_host/;
     proxy_pass        http://127.0.0.1:$tunnel_port;
 
     access_log /var/log/nginx/tunnel.log;
   }
 }
 
+{% for service in services %}
```

Comment:
Generate an nginx `server` configuration for each service.

Right now it's just `callrail` and `swappy`.